custom/enikshay/integrations/nikshay/repeaters.py

```diff
@@ -128,6 +134,49 @@ def allowed_to_forward(self, episode_case):
         )
 
 
+class NikshayFollowupRepeater(CaseRepeater):
+    followup_for_tests = ['end_of_ip', 'end_of_cp']
+
+    class Meta(object):
+        app_label = 'repeaters'
+
+    include_app_id_param = False
+    friendly_name = _("Forward eNikshay Patient's Follow Ups to Nikshay")
```

Comment:
Yes. I think it would be great to document in the UI which case types this expects (this is so that we don't have to override the `new` view to always have the right case type).